src/scribe_data/cli/main.py

```diff
@@ -245,21 +282,14 @@ def main() -> None:
         total_wrapper(args.language, args.data_type, args.all)
 
     elif args.command in ["convert", "c"]:
-        if args.output_type in ["csv", "tsv"]:
-            convert_to_csv_or_tsv(
-                args.language,
-                args.data_type,
-                args.output_dir,
-                args.overwrite,
-            )
-
-        elif args.output_type == "sqlite":
-            convert_to_sqlite(
-                args.language,
-                args.data_type,
-                args.output_dir,
-                args.overwrite,
-            )
+        convert(
```

Comment:
Wonderful :)

Reply:
Functioning just like the others now 😊 I'll do a rename to `convert_wrapper` for consistency and so people know this functions conditionally to call sub functions. More of a switcher, but then `total_switcher` sounds a bit weird.